ros/api/v0/hosts.py

```diff
@@ -151,11 +152,12 @@ def build_sort_expression(self, order_how, order_method):
 class HostDetailsApi(Resource):
     profile_fields = {
         'host_id': fields.String(attribute='inventory_id'),
-        'performance_record': fields.String,
-        'display_performance_score': fields.String,
         'rating': fields.Integer,
-        'recommendation_count': fields.Integer,
```

Comment:
Any reason you not included keys - `display_performance_score, recommendation_count, state` in response?

Reply:
Yeah, so I tried to keep the details keys consistent with this [mockup](https://marvelapp.com/prototype/9408d9j/screen/74816603). If we still want all other fields related to a host, we can include those. 
I just didn't find much use of that data as we are not showing the same on  details page UI. (If we are to consume this API for the details page).
Let me know if we still want those keys.